### ISSUE TITLE
Is --no-check-certificate necessary?

### ISSUE THREAD

**Issue by dxgldotorg** [bigtreetech/OctoBTT#42]
The assets are downloaded from raw.githubusercontent.com which has valid SSL certificates; if a certificate error occurs that means there is something wrong.

Or is the "Great Firewall" damaging your connections?  Because those in the rest of the world can probably omit that command line parameter safely.

**Comment by bigtreetech** [comment]
However, it worked fine in our installation tests, unless it was caused by a network failure, if you failed to run, you can try to cancel the run parameter.

**Comment by dxgldotorg** [comment]
Adding --no-check-certificate bypasses the security features of HTTPS, and if it isn't required should be omitted.

**Comment by Shine6Z** [comment]
Of course, in order to ensure that everyone can use it normally, I don't think this option should be cancelled, because there is no reason for this option to harm the interests of users. If there is, I will change other methods to ensure the safety of users. If you have a better way, I will introduce it into the project after testing confirmation.
